<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>foodInformation</class>
 <widget class="QWidget" name="foodInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>490</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>foodInformation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="4" column="0" rowspan="2">
    <widget class="QFrame" name="frame_lcd">
     <property name="minimumSize">
      <size>
       <width>321</width>
       <height>101</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLCDNumber" name="lcd_number">
        <property name="minimumSize">
         <size>
          <width>311</width>
          <height>81</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_g">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>g</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btn_back">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>51</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>249</red>
           <green>249</green>
           <blue>249</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>214</red>
           <green>214</green>
           <blue>214</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>249</red>
           <green>249</green>
           <blue>249</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>214</red>
           <green>214</green>
           <blue>214</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>249</red>
           <green>249</green>
           <blue>249</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>214</red>
           <green>214</green>
           <blue>214</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>214</red>
           <green>214</green>
           <blue>214</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_title">
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Food Item Information</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="grpBx_nutriInfo">
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>298</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_energy">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Energy (kcal)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_evergyVal">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_protein">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Protein (g)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_proteinVal">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_sugar">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sugar (g)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_sugarVal">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_fibre">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fibre (g)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_fibreVal">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_fat">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fat (g)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lbl_fatVal">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_salt">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Salt (g)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lbl_saltVal">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QPushButton" name="btn_new">
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>110</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lbl_foodPic">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn_tare">
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tare</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_suggestion">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>Incorrect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_addIntake">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Correct</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
